charts/rancher-turtles/values.yaml

```diff
@@ -3,3 +3,5 @@ rancherTurtles:
   tag: v0.0.0
   imagePullPolicy: Never
   namespace: rancher-turtles-system
+capiOperator:
```

Comment:
Would it also be worth exposing some of the values of the CAPI operator? That way when we install Rancher Turtles we can also say which providers to install (or not install cert-manager)  for example

Reply:
Helm supports passing arguments to dependencies, you can do `--set cluster-api-operator.enabled=true`